packages/nx/src/utils/command-line-utils.ts

```diff
@@ -11,7 +11,6 @@ export interface RawNxArgs extends NxArgs {
 }
 
 export interface NxArgs {
-  target?: string;
```

Comment:
This property is explicitly deleted in `splitArgsIntoNxArgsAndOverrides`, it shouldn't exist anymore and removing it would have caused a build error and prevented this bug from occurring:

```typescript
  if (args['target'] && !args['targets']) {
    args['targets'] = [args['target']];
  }
  delete args['target'];
  delete args['t'];
```